core/invoker/src/main/scala/whisk/core/containerpool/docker/DockerContainer.scala

```diff
@@ -120,7 +122,7 @@ object DockerContainer {
  * @param id the id of the container
  * @param ip the ip of the container
  */
-class DockerContainer(id: ContainerId, ip: ContainerIp)(
+class DockerContainer(val id: ContainerId, val ip: ContainerIp)(
```

Comment:
Do these need to be public?

Reply:
id/ip are needed by the HTTP API during initialize() and run() - latest changes have these as protected val, but also has the HTTP API methods initialize() and run() moved into a separate class so they can be shared - see ContainerApi.scala